homeassistant/components/godice/manifest.json

```
@@ -0,0 +1,18 @@
+{
+  "domain": "godice",
+  "name": "godice",
+  "bluetooth": [
+    {
+      "local_name": "GoDice*"
+    }
+  ],
+  "codeowners": ["@anton-ptashnik"],
+  "config_flow": true,
+  "dependencies": ["bluetooth_adapters"],
+  "documentation": "https://www.home-assistant.io/integrations/godice",
+  "homekit": {},
+  "iot_class": "local_push",
+  "requirements": ["godice==0.1.0"],
+  "ssdp": [],
+  "zeroconf": []
```

```suggestion
  "iot_class": "local_push",
  "requirements": ["godice==0.1.0"],
```

Empty keys can be removed

---

addressed, thanks for the hint!